src/workspacer.FSharpConfig/Extensions.cs

```diff
@@ -0,0 +1,67 @@
+﻿namespace workspacer.FSharpConfig
+{
+    using System;
+    using System.Collections.Generic;
+    using System.IO;
+    using System.Linq;
+    using System.Reflection;
+
+    public static class Extensions
```

Comment:
Since this class extends the `AppDomain`-class, I think a better name for it would be `AppDomainExtensions`.